cardano-crypto-class/cardano-crypto-class.cabal

```diff
@@ -31,7 +31,6 @@ common project-config
                         -Wincomplete-uni-patterns
                         -Wpartial-fields
                         -Wredundant-constraints
```

Comment:
I would suggest leaving these things in place; these warning flags are there for a purpose, and I don't think there is a good reason to accept unused packages in a release build here.

You can use cabal.project.local to override things for local development.

Reply:
Sorry, what happened with these? Did I accidentally remove them?

Reply:
I think he meant to comment on the line below. You removed `-Wunused-packages`.